clang/test/CIR/CallConvLowering/varargs.c

```diff
@@ -0,0 +1,17 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -fclangir-call-conv-lowering -emit-cir-flat -mmlir --mlir-print-ir-after=cir-call-conv-lowering %s -o %t.cir
```

Comment:
Awesome! Can  you put this under the x86_64 dir? Or Aarch64 if you prefer changing the triple?

Reply:
well, the idea was to place something that is really target independent in the root dir. But I moved them )

Reply:
I understand your intent, and even though you're coming from "I fixed target independent code" CIR is not really target independent, so best to put this test with their closest friends. Thanks for the fix.